website/docs/docs/develop/dbt-cloud-features.md

```diff
@@ -3,42 +3,74 @@ title: "dbt Cloud features"
 id: "dbt-cloud-features"
 ---
 
+## dbt Cloud IDE
```

Comment:
@runleonarun : just a heads up since you own git issue #2074 (dbt Cloud features page). mirna has populated it with IDE-specific content